examples/cpp/hello_world/HelloWorld_controller_profile.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<profiles xmlns="http://www.eprosima.com" >
+    <participant profile_name="hello_world_controllers_participant_profile" is_default_profile="true">
+        <domainId>0</domainId>
+        <rtps>
+            <name>hello_world_controllers_participant</name>
+            <builtinTransports>UDPv4</builtinTransports>
+        </rtps>
+    </participant>
+    <data_writer profile_name="hello_world_controllers_dataWriter_profile" is_default_profile="true">
+        <qos>
+            <durability>
+                <kind>TRANSIENT_LOCAL</kind>
+            </durability>
+            <reliability>
+                <kind>RELIABLE</kind>
+            </reliability>
+        </qos>
+        <topic>
+            <historyQos>
+                <kind>KEEP_LAST</kind>
+                <depth>100</depth>
+            </historyQos>
+            <resourceLimitsQos>
+                <max_samples>100</max_samples>
+                <max_instances>1</max_instances>
+                <max_samples_per_instance>100</max_samples_per_instance>
+            </resourceLimitsQos>
```

Comment:
I don't think this is what I'd do for a controller

Reply:
Removed